game/scripts/npc/items/custom/item_dagger_of_moriah.txt

```diff
@@ -78,7 +78,7 @@
       "02"
       {
         "var_type"                                        "FIELD_INTEGER"
-        "sangromancy_self_damage"                         "100 100"
+        "sangromancy_self_damage"                         "75 100"
```

Comment:
second level shouldn't really be worse to use than the first level. I would put both at 75